src/Sylius/Bundle/ShippingBundle/Resources/config/serializer/Model.ShippingCategory.yml

```diff
@@ -6,24 +6,32 @@ Sylius\Component\Shipping\Model\ShippingCategory:
             expose: true
             type: integer
             xml_attribute: true
+            groups: [Default, Details]
         name:
             expose: true
             type: string
+            groups: [Default, Details]
+        code:
+            expose: true
+            type: string
+            groups: [Default, Details]
         description:
             expose: true
             type: string
+            groups: [Details]
         createdAt:
             expose: true
             type: DateTime
+            groups: [Details]
         updatedAt:
             expose: true
             type: DateTime
-        code:
-            expose: true
-            type: string
+            groups: [Details]
     relations:
         - rel: self
           href:
-                route: sylius_api_shipping_category_show
-                parameters:
-                    id: expr(object.getId())
+            route: sylius_api_shipping_category_show
+            parameters:
+                id: expr(object.getId())
+            exclusion:
+                groups: [Details]
```

Comment:
To be honest I do not know what is the recommendation here. Have you done some research?